content/api/authentication/systemuserapi/clientdelegation/_index.nb.md

```
@@ -20,26 +24,33 @@ For testmiljø, legg til query-parameteren ?test=true.
 
 ### Scopes
+
 Idporten token utvekslet som altinn token med scope <mark>altinn:clientdelegations.read</mark>
 
 ### Innholdstype
+
 application/json
 
 ## Query parametere
 
 #### party
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)